frontend/src/my-books/myBookModal/ShelfModal.css

```diff
@@ -0,0 +1,77 @@
+.shelf-modal-container {
+    display:flex;
+    flex-direction:column;
+    text-align: left;
+    padding-top: 32px;
+    padding-bottom: 32px;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  }
+  
+  .modal-title {
+    color:black;
+    font-size: 20pt;
+    letter-spacing: -0.5px;
+    font-weight: bold;
+  }
+  
+  .shelf-modal-desc-container{
+    font-size: 10pt;
+    padding-top: 40px;
+    padding-bottom: 60px;
+  } 
+  
+  .PrivateHiddenCss-smDown-44{
+    display: flex;
+    align-items:center;
+    justify-content: space-between;
+  }
+
+  .shelf-modal-desc-container > .PrivateHiddenCss-smDown-44 > p{
+      width: 30%;
+      margin-right: 15px;
+  }
+
+  .shelfInput > .MuiOutlinedInput-root{
+    width: 100%;
+    margin-right: 600px; 
+    border-radius: 20px;
+  }
+  
+  .shelf-button-container{
+      display:flex;
+      justify-content:space-between;
+  }
+  .shelf-modal-button {
+    width: 100px ;
+  }
+
+
+  @media screen and (max-width: 959px){
+    .shelfInput > .MuiOutlinedInput-root{
+        width: 100%;
+        margin-right: 600px; 
+        border-radius: 20px;
+      }
+      .PrivateHiddenCss-mdUp-46{
+        display: flex;
+        align-items:center;
+        justify-content: space-between;
+      }
+  }
+
+  @media screen and (max-width: 420px) {
+    .shelf-modal-container {
+        text-align: center;
+    }
+    .shelf-button-container{
+        display:flex;
+        flex-direction: column;
+        /* align-items:center; */
+    }
+    .shelf-modal-button {
```

Comment:
Let's add some space in between the /shelf-modal-buttons for mobile. Right now they are displayed as a column in mobile which great! We just need to push them apart about 24-26px(let's try that).